File: code/JL_OTA_SDK/otasdk_java/src/main/java/com/jieli/otasdk_java/base/BaseDialogFragment.java
package com.jieli.otasdk_java.base;

import androidx.fragment.app.DialogFragment;
import androidx.fragment.app.FragmentManager;

/**
 * @ClassName: BaseDialogFragment
 * @Description: java类作用描述
 * @Author: ZhangHuanMing
 * @CreateDate: 2021/12/23 9:33
 */
public class BaseDialogFragment extends DialogFragment {
    public void show(FragmentManager manager) {
        if (manager != null) {
            show(manager, getClass().getCanonicalName());
        }
    }
}

File: code/JL_OTA_SDK/otasdk/src/main/java/com/jieli/otasdk/base/BaseDialogFragment.kt
package com.jieli.otasdk.base

import androidx.fragment.app.DialogFragment
import androidx.fragment.app.FragmentManager

/**
 ************************************
 *@Author revolve
 *创建时间：2019/7/31  15:44
 *用途
 ************************************
 */
open class BaseDialogFragment : DialogFragment() {


    fun show(manager: FragmentManager?) {
        manager?.let { super.show(it, javaClass.canonicalName) }
    }

}